core/executor.py

```
@@ -47,644 +52,215 @@ def __init__(
-        Args:
-            task: 用户任务
-            routing_decision: 路由决策
-            spec_initializer: Spec初始化器
-
-        Returns:
-            Manifest对象
-        """
-        if not spec_initializer:
-            logger.warning("No spec_initializer provided, skipping spec initialization")
+            return self.manifest_manager.load()
+        except:
```

Ruff (0.15.9)

[error] Do not use bare `except`

(E722)